### PR TITLE
Add is_locked field

### PR DESCRIPTION
**Pull request by Formulka**

### CODE REVIEW

File: pymess/locale/cs/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -408,127 +411,133 @@ msgstr "vztažený objekt zprávy dialeru"
...
 msgstr "Poslední webhook přijat"
 
-#: models/emails.py:54
+#: models/emails.py:74
 msgid "info changed at"
 msgstr "informace změněny"
 
-#: models/emails.py:61 models/emails.py:81 models/emails.py:108
+#: models/emails.py:80
+#, fuzzy
```
Comment (matllubos) [inline pymess/locale/cs/LC_MESSAGES/django.po:459]: there should not be a fuzzy

File: pymess/locale/cs/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -274,31 +270,39 @@ msgstr "tag"
...
 
-#: models/common.py:168
+#: models/common.py:170
 msgid "is active"
 msgstr "je aktivní"
 
-#: models/common.py:170
+#: models/common.py:171
+msgid "is locked"
+msgstr "je zamčeno"
```
Comment (matllubos) [inline pymess/locale/cs/LC_MESSAGES/django.po:295]: je uzamčeno?

Comment (Formulka) [reply pymess/locale/cs/LC_MESSAGES/django.po:295]: you are right, that sounds better

File: pymess/locale/cs/LC_MESSAGES/django.po (hunk 1 above)
Comment (Formulka) [reply pymess/locale/cs/LC_MESSAGES/django.po:459]: sorry, forgot to delete it

**Pull request merged**